asdf/tests/schema_tester.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, schema_path, parent, example):
     def _find_standard_version(self):
         filename = self.filename
         components = filename[filename.find('schemas') + 1:].split(os.path.sep)
+        tag = "tag:" + ":".join(["{}" for i in range(len(components[1:]))])
         tag = 'tag:{}:{}'.format(components[1], '/'.join(components[2:]))
```

Comment:
Should this line be deleted?

Reply:
Actually, this is not correct either. I'm going to close this.